frontend/src/forms/ScamInfoForm.js

```diff
@@ -138,26 +138,22 @@ export const ScamInfoForm = ({ onSubmit }) => (
             </Text>
 
             <Container
-              width="305px"
-              marginTop={[3, null, 4]}
+              width="300px"
```

Comment:
I’m not sure exactly why, but the 305 width was needed to make the button the same width as the 300 pixel text box above it.